### DISCUSSION TITLE
ESP-CAM + GY-21 HTU21 Feuchtigkeit und Temperatur Sensor

### DISCUSSION THREAD

**Discussion by Sub-7** [jomjol/AI-on-the-edge-device#2163] [Q&A]
Gibt es eine Möglichkeit Feuchtigkeit und Temperatur Sensor parallel mit der Cam zu betreiben?
Wäre ganz toll.

![Screenshot 2023-03-12 at 08-59-31 GY-21 HTU21 Feuchtigkeit und Temperatur Sensor](https://user-images.githubusercontent.com/30667941/224532048-6c80dc23-6b87-41c0-842f-0ce0f386e6f7.png)

**Comment by caco3** [comment]
nein, das geht nicht, weil zuwenig freie Pins zur Verfügung stehen.
